src/styles/blocks/card.css

```diff
@@ -0,0 +1,70 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  box-sizing: content-box;
+  height: 134px;
+  width: 160px;
+  padding: 0 3px;
```

Comment:
Strange solution, better to use margin auto, or flex to align, instead of hardcoded 3px paddings